addOns/oast/src/main/java/org/zaproxy/addon/oast/services/interactsh/InteractshService.java

```diff
@@ -176,14 +176,26 @@ public void register() throws InteractshException {
             HttpMessage reqMsg = new HttpMessage(reqHeader, reqBody);
             httpSender.sendAndReceive(reqMsg);
             if (reqMsg.getResponseHeader().getStatusCode() != 200) {
+                LOGGER.warn(
+                        "Error during interactsh register, due to bad HTTP code {}. Content: {}",
+                        reqMsg.getResponseHeader().getStatusCode(),
+                        reqMsg.getResponseBody().toString());
```

Comment:
Remove the `toString()`, let the string representation be created when really needed.